src/main/kotlin/com/shortener/controller/UrlController.kt

```diff
@@ -0,0 +1,49 @@
+package com.shortener.controller
+
+import com.shortener.dto.UrlShortenRequest
+import com.shortener.service.UrlService
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Controller
+import org.springframework.ui.Model
+import org.springframework.ui.set
+import org.springframework.validation.BindingResult
+import org.springframework.web.bind.annotation.*
+import java.net.URI
+
+
+@Controller
+class UrlController(private val urlService: UrlService) {
+
+    @GetMapping
+    fun getEncodeUrlPage(model: Model): String {
+        model.addAttribute("urlShortenRequest", UrlShortenRequest())
+
+        //TODO: remove it (displays the shortened url history)
+        model["allEncodedUrls"] = urlService.getAllShortenedUrls()
+
+        return "encode-url"
```

Comment:
Magic string. perhaps use a const instead?

Reply:
Extracted to a constant